### PR TITLE
Staff page redesign

### PR DESCRIPTION
**Pull request by PradyumnaShome**
- **Staff page** - Switched to a flexbox as opposed to three images at a time in a loop via jekyll templating, as it is more responsive (adapts to viewport), and creates an actual grid. Moved the names to below the image to avoid covering the image from the bottom.
- Changed sorting to be based on first name, not by NetID, which is natural
- Ignore the color change in the screenshot.
(Before diff)
image
(After diff)
image

### CODE REVIEW

**Comment by redsn0w422** [comment]
@PradyumnaShome can you also include screenshots of desktop before and after? Thanks!

**Comment by PradyumnaShome** [comment]
*Desktop Before*
Screenshot_64

*Desktop After*
Screenshot_66

File: css/main.css [hunk 1]
```diff
@@ -1,15 +1,17 @@
 :root {
   --color-primary: #2c72ad;
-  --color-secondary: #2f5fffdc;
+  --color-secondary: rgb(53, 144, 219);
```
Comment (redsn0w422) [inline css/main.css]: These should be consistent - either use all hex or all RGB.

File: css/main.css [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
+  --color-emphasis: rgb(0, 129, 214);
```
Comment (redsn0w422) [inline css/main.css]: Same here.

File: css/main.css [hunk 3]
```diff
@@ -591,3 +559,54 @@ li.navbar-link:hover a, li.navbar-sublink:hover {
...
+.staff-container:hover div.staff-image {
+    box-shadow:0 4px 12px 0 var(--color-translucent), 0 6px 20px 0 var(--color-translucent);
+}
+
+.staff-name {
+    bottom: 0px;
+    width: 100%;
+    text-align: center;
+    font-size: 1.1em;
+    padding-top: 2vh;
```
Comment (redsn0w422) [inline css/main.css]: Can you increase the spacing between the photo and the name? It looks a bit cramped at the moment.

File: css/main.css [hunk 4]
```diff
@@ -591,3 +559,54 @@ li.navbar-link:hover a, li.navbar-sublink:hover {
...
+    width: 22vh;
+    height: 22vh;
+    overflow: hidden;
+    background-size: cover;
+    padding: 0;
+    border-radius: 50%;
+}
+
+.staff-container:hover div.staff-image {
+    box-shadow:0 4px 12px 0 var(--color-translucent), 0 6px 20px 0 var(--color-translucent);
```
Comment (redsn0w422) [inline css/main.css]: What does this do?

File: css/main.css (hunk 1 above)
Comment (PradyumnaShome) [reply css/main.css]: Sure!

File: css/main.css (hunk 4 above)
Comment (PradyumnaShome) [reply css/main.css]: Adds a shadow when you hover over someone's photo / text.
Like this: 
image

File: css/main.css (hunk 3 above)
Comment (PradyumnaShome) [reply css/main.css]: Done.

**Comment by PradyumnaShome** [comment]
Now
image

**Comment by bhuvy2** [comment]
Sister issue https://github.com/illinois-cs241/illinois-cs241.github.io/pull/109#issuecomment-516923419

**Pull request merged**